lib/themes/dosomething/paraneue_dosomething/js/onboarding.js

```diff
@@ -13,9 +13,10 @@ const ReactDom = require('react-dom');
 $(document).ready(function() {
   const campaign = Drupal.settings.dsCampaign || null;
   const user = Drupal.settings.dsUser || null;
+  const disabled_langs = ['mx', 'br']
   let localStorageKey = null;
 
-  if (campaign && user) {
+  if (campaign && user && disabled_langs.indexOf(campaign.language.language_code) == -1) {
```

Comment:
Also does this mean the on-boarding experience will only ever be disabled by that local storage key if the user has one of the language codes? Should the conditional be `|| isDisabledForLanguageCode` instead?